src/drivers/websql.js

```diff
@@ -39,14 +39,15 @@ function _initStorage(options) {
 
         // Create our key/value table if it doesn't exist.
         dbInfo.db.transaction(function(t) {
-            t.executeSql('CREATE TABLE IF NOT EXISTS ' + dbInfo.storeName +
-                         ' (id INTEGER PRIMARY KEY, key unique, value)', [],
-                         function() {
-                self._dbInfo = dbInfo;
-                resolve();
-            }, function(t, error) {
-                reject(error);
-            });
+            t.executeSql(
+                'CREATE TABLE IF NOT EXISTS ' + dbInfo.storeName +
```

Comment:
I've logged #740 because eventually it'd be cool to get away from this kind of string concats.

Reply:
Tried that on many places, but oneLine will be superuseful for the rest 👍